inc/class-db.php

```diff
@@ -12,6 +12,8 @@
 use QM_Backtrace;
 use WP_Error;
 
+const SAVEQUERIES = true;
```

Comment:
Where is `SAVEQUERIES` normally defined? Note that the error message isn't saying that the constant `SAVEQUERIES` _should_ exist in this namespace - normally it would be defined using `define()` somewhere globally. We need to understand and account for the situation where that isn't the case rather than just define it here.

We wouldn't necessarily want `SAVEQUERIES` to be `true` in production either for example.

Reply:
SAVEQUERIES is usually defined in wp-config.php, sometimes forced by tools like Query Montior in db.php. Anything later than that would be too late.